mbed_lstools/main.py

```diff
@@ -170,6 +176,8 @@ def get_mbedls_version():
     mbeds.debug(__name__, "mbed-ls ver. " + get_mbedls_version())
     mbeds.debug(__name__, "host: " +  str((mbed_lstools_os_info())))
 
+    mbeds.list_unmounted = opts.list_unmounted
```

Comment:
This feels strange, no other option in the CLI modifies the `mbeds` object directly.

Instead, can this option be passed into the `list` functions directly as an argument?

Reply:
Could you remove this assignment?